haminfo/mqtt/filters.py

```
@@ -0,0 +1,227 @@
+                with self.stats_lock:
+                    self.stats['report_counter'] = (
+                        self.stats.get('report_counter', 0) + 1
+                    )
+                return True  # Signal success
+            else:
+                return None
+        except ValueError as ex:
+            self.session.rollback()
+            logger.error(f'Invalid weather report: {ex}')
+            return None
+        except Exception as ex:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)